topology/University of Michigan/AGLT2/AGLT2.yaml

```
@@ -33,6 +33,41 @@ Resources:
         Description: Generic squid service
     VOOwnership:
       ATLAS: 90
+AGLT2-squid:
```

```suggestion
AGLT2-squid-2:
```

Choosing a new resource name